pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1533,6 +1533,58 @@ var _ = Describe("Converter", func() {
 			Expect(domainSpec.Devices.Rng).ToNot(BeNil())
 		})
 
+		When("QEMU CBios debug logs flag is not defined in VMI labels", func() {
+			It("should not enable QEMU CBios debug logs", func() {
+
+				domain := api.Domain{}
+				err := Convert_v1_VirtualMachine_To_api_Domain(vmi, &domain, c)
+				Expect(err).To(BeNil())
+
+				if domain.Spec.QEMUCmd != nil && (domain.Spec.QEMUCmd.QEMUArg != nil) {
+					Expect(domain.Spec.QEMUCmd.QEMUArg).ShouldNot(Or(HaveKey("-chardev"), HaveKey("-device")))
```

Comment:
@iholder-redhat Can we use a "stronger" identifier here in the expect? for example that the key should not caontain ```isa-debugcon,iobase=0x402```?

Reply:
The problem is that QEMUArg is a slice, not a map, therefore there is no key/value relationship.
I tried to find Gomega API to validate that QEMUArg contains a "sub-slice" with all four "key/value" arguments, but unfortunately couldn't find one.

I will add validation for the "values" to be part of QEMUArg without validating right order. I will be happy to hear better approach for this.